tests/test_serialization.py

```diff
@@ -220,3 +220,12 @@ def test_region_only_pagexml_serialization_validation(self):
 
         fp.write(serialization.serialize([], image_name='foo.png', template='pagexml', regions=self.bl_regions))
         validate_page(self, fp)
+
+    def test_serialize_segementation_alto(self):
```

Comment:
Could you add a test for PageXML as well? And there's a typo in the function name.

Reply:
Added a test for PageXML and fixed the typo in https://github.com/mittagessen/kraken/pull/329/commits/48d93a606b7b9eaad76518e34e557640dd6a9e7d